### ISSUE TITLE
map info

### ISSUE THREAD

**Issue by druni** [GAMMACASE/ServerRedirect#14]
Please add map info to advertisement in chat and in the !servers menu

Map is the most important part about changing server.

In the menu it could look like [64/64] de_dust2 [SERVER NAME]

Same with chat, instead of showing IP in the chat you could show the current map.

IP is irrelevant...

**Comment by GAMMACASE** [comment]
Yeah that is definitely in my todo list. Also for server adverts in chat I'll probably do a cvar where you can choose what to display and you can tweak this for your liking.
